### PR TITLE
feature/#29-new-finding-azure-cache-no-network-acl

### PR DESCRIPTION
**Pull request by misg**
This PR adds support for the Redis Cache service.
Please look at the [proprietary PR](https://github.com/nccgroup/ScoutSuite-Proprietary/pull/76) as well.

### CODE REVIEW

File: ScoutSuite/output/data/html/partials/azure/services.rediscache.caches.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+</script>
+
+<script>
+    Handlebars.registerPartial("services.rediscache.caches", $("#services\\.rediscache\\.caches\\.partial").html());
+</script>
+
+<!-- Single Redis Cache template -->
+<script id="single_rediscache_caches-template" type="text/x-handlebars-template">
+    {{> modal-template template='services.rediscache.caches' }}
```
Comment (Aboisier) [inline ScoutSuite/output/data/html/partials/azure/services.rediscache.caches.html]: 💯

File: requirements.txt [hunk 2]
```diff
@@ -24,5 +24,6 @@ oauth2client>=4.1.3
 azure-mgmt-storage>=3.0.0
 azure-mgmt-monitor
 azure-mgmt-sql
+azure-mgmt-redis
```
Comment (j4v) [inline requirements.txt]: Please start setting minimum versions for all libraries.

**Comment by j4v** [comment]
Just to make sure, are there CIS checks for this service? If not please put the HTML partial(s) and python code into the proprietary repo.

File: ScoutSuite/output/data/html/partials/azure/services.rediscache.caches.html (hunk 1 above)
Comment (misg) [reply ScoutSuite/output/data/html/partials/azure/services.rediscache.caches.html]: :fireworks:

File: requirements.txt (hunk 2 above)
Comment (misg) [reply requirements.txt]: Done!

**Comment by misg** [comment]
> Just to make sure, are there CIS checks for this service? If not please put the HTML partial(s) and python code into the proprietary repo.

Nice catch, my bad! I made the HTML partial and python code private.

**Pull request merged**